libs/metrics/base.py

```diff
@@ -325,6 +328,7 @@ def get_daily_report(cls, entities: List[str], from_date: date, to_date: date) -
         base_directory = cls.get_entity_base_directory()
         date_range = pd.date_range(start=from_date, end=to_date)
         base_results = {}
+        daily_headers = [h for h in cls.csv_headers if h not in cls.ignored_headers]
```

Comment:
Maybe you can include the following property
```
@property
def report_headers(self):
         return [h for h in cls.csv_headers if h not in cls.ignored_headers]
```
and use it instead fd repeating the same logic.